public/actions.js

```diff
@@ -605,6 +605,45 @@ export const StartTimer = (state, { timerStartedAt, timerDuration }) => [
   }),
 ];
 
+export const Skip = (state, { timerDuration, timerStartedAt = null }) => {
+  console.log("action - Next");
```

Comment:
Let's remove this console.log

Reply:
Thanks for catching. Left over when I was trying to understand a few things about the system.